src/lib/storage/storage_manager.hpp

```diff
@@ -19,7 +18,7 @@ class AbstractLQPNode;
 
 // The StorageManager is a singleton that maintains all tables
 // by mapping table names to table instances.
-class StorageManager : public Singleton<StorageManager> {
+class StorageManager {
```

Comment:
For all classes that were previously singleton, we should consider inheriting from Noncopyable. We don't want to end up with two StorageManagers by accident.